From: Marcel Holtmann <marcel@holtmann.org>
Date: Sun, 13 Oct 2013 09:49:57 -0700
Subject: [PATCH] Bluetooth: Access BNEP session addresses through L2CAP
 channel
Patch-mainline: v3.13-rc1
Git-commit: 65f53e9802dbfae0e5758a91793c3f5f8bece49b (partial)
References: bsc#1071470 CVE-2017-15868

The L2CAP socket structure does not contain the address information
anymore. They need to be accessed through the L2CAP channel.

Signed-off-by: Marcel Holtmann <marcel@holtmann.org>
Signed-off-by: Johan Hedberg <johan.hedberg@intel.com>
Acked-by: Cho, Yu-Chen <acho@suse.com>
---
 net/bluetooth/bnep/core.c |    1 +
 1 file changed, 1 insertion(+)

--- a/net/bluetooth/bnep/core.c
+++ b/net/bluetooth/bnep/core.c
@@ -49,6 +49,7 @@
 #include <asm/unaligned.h>
 
 #include <net/bluetooth/bluetooth.h>
+#include <net/bluetooth/l2cap.h>
 #include <net/bluetooth/hci_core.h>
 #include <net/bluetooth/l2cap.h>
 
